examples/ppo_config_cnn_daily.yml

```diff
@@ -0,0 +1,57 @@
+train:
+  seq_length: 612
+  epochs: 100
+  total_steps: 100000
+  batch_size: 12
+
+  checkpoint_interval: 10000
+  eval_interval: 500
+
+  pipeline: "PromptPipeline"
+  orchestrator: "PPOOrchestrator"
+  trainer: "AcceleratePPOTrainer"
+
+model:
+  model_path:  "google/flan-t5-large" #  "lvwerra/gpt2-imdb" #"lvwerra/gpt2-imdb"
+  model_arch_type: "encoder_decoder"
+  tokenizer_path:  "google/flan-t5-large" #  "lvwerra/gpt2-imdb" #"google/flan-t5-large" # "lvwerra/gpt2-imdb"
+  num_layers_unfrozen: 2
```

Comment:
Remove the comments in the `*_path`s

Reply:
Will remove it in next updates. Thank you.